learning/basics/task1/MT-input_print.py

```diff
@@ -0,0 +1,24 @@
+# '''
+# Create new file "<initials>-<task_name>" in this directory, there you should solve this task.
+# This file should stay unchanged! Example: "AW-input_print.py"
+# Create a script that will ask about name, and two numbers tu multiply.
+# Script should display greetings and result of multiplication.
+# Example:
+#     out: Hello! What is your name?
+#     > Artur
+#     out: Hello Artur, give me first number to multiply!
+#     >5
+#     out: now give me second number.
+#     >3
+#     out: Artur, your result of multiplication for 3 and 5 is 15!
+# IMPORTANT: Learn about f-strings in python, use them to print messages!
+# '''
+print("Hello, what is your name?")
+name = input("")
+print("Hi {name} put first number to multiply".format(name=name))

```

Comment:
This solution is perfectly fine, but much more popular way to do this is f-string. Look at example below, it looks cleaner
```suggestion
print(f"Hi {name} put first number to multiply")

```